runtime/lua/vim/filetype.lua

```diff
@@ -2230,29 +2293,36 @@ local function compare_by_priority(a, b)
   return a[next(a)][2].priority > b[next(b)][2].priority
 end
 
---- @param t vim.filetype.mapping
+local parse_pattern = function(pat)
```

Comment:
```suggestion
--- @param pat string
local function parse_pattern(pat)
```

Reply:
Addressed 168d9e07fd855a2ef33bbe922b2295d9a9e7c3dd (with even and a bit more :) )